### PR TITLE
[issue #671/#672] remediation and OVAL for bootloader_nousb_argument

### LINKED ISSUE #671: OVAL for RHEL7 bootloader_nousb_argument

### PR DESCRIPTION
**Pull request by shawndwells**
Added remediation for bootloader_nousb_argument into RHEL6/7/fedora

Added OVAL for bootloader_no_usb argument into rhel7/fedora

Resolves #671 and #672

### CODE REVIEW

File: shared/fixes/bash/bootloader_nousb_argument.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+USB_KEYBOARD=$(grep 'Product=' /proc/bus/usb/devices 2>/dev/null| egrep -ic '(ps2 to usb adapter|keyboard|kvm|sc reader)')
+if [ "${USB_KEYBOARD}" = "0" ]; then
+	/sbin/grubby --update-kernel=ALL --args="nousb"
```
Comment (mpreisler) [inline shared/fixes/bash/bootloader_nousb_argument.sh]: Is `--args` persistent or will it get ignored when new kernel is shipped? I think `--args` is a one-off option and will be ignored but I am not sure.

**Comment by redhatrises** [comment]
I wonder if this should be added to the `GRUB_CMDLINE_LINUX` in `/etc/default/grub` instead of running `grubby`?

**Comment by mpreisler** [comment]
@redhatrises Yeah I think that's the case.

**Comment by iankko** [comment]
IMHO in ideal case it should be updating both:
* the ```/etc/default/grub``` for static configuration, and run
* ```grubby``` to update existing (already installed) kernel parameters

Something like already existing RHEL-7 remediation for ```bootloader_audit_argument``` is doing:
&nbsp; &nbsp; [1] https://github.com/OpenSCAP/scap-security-guide/blob/master/RHEL/7/input/fixes/bash/bootloader_audit_argument.sh

Should we in the future want to have static & runtime setting split, that remediation can be easily split into two scripts.

**Comment by iankko** [comment]
I will keep this open (to be handled within 0.1.28 milestone). The conflicts && related necessary changes aren't that big, this couldn't be fixed yet.

**Comment by iankko** [comment]
This is now replaced by https://github.com/OpenSCAP/scap-security-guide/pull/1034

Feel free to review that one instead of this. Closing this one.

**Pull request closed without merging**